ui/package.json

```diff
@@ -101,7 +101,7 @@
     "ember-template-lint": "^2.9.1",
     "ember-test-selectors": "^5.0.0",
     "ember-truth-helpers": "^2.0.0",
-    "eslint": "^7.5.0",
+    "eslint": "^8.4.0",
```

Comment:
Generally, I don't think dependabot PR's are a good way to update Ember apps. We're better off running:  `yarn upgrade interactive` and following a process around upgrading.

Reply:
Yeah that's totally fine by me... we can keep dependabot just to warn us when there's something security sensitive we're looking at and otherwise toss the PRs if you want.